pkg/node/Dockerfile-kernel-5.4.build

```diff
@@ -3,7 +3,4 @@ FROM    ubuntu:21.04
 ADD     health_probe    health_probe
 
 # On Ubuntu 21.04 fdisk is not installed by defaul
-# Remove bash packet to get rid of related CVEs
-RUN     apt update --no-install-recommends -y -q; apt remove --no-install-recommends -y --allow-remove-essential -q bash; apt install --no-install-recommends -y -q util-linux parted xfsprogs lvm2 fdisk gdisk strace udev net-tools
-
-
+RUN     apt update --no-install-recommends -y -q; apt install --no-install-recommends -y -q util-linux parted xfsprogs lvm2 fdisk gdisk strace udev net-tools
```

Comment:
Should we place comment here about unstable bash version?

Reply:
What version is unstable? There is no hardcoded version.

Reply:
I'm not sure, why did we delete it?

Reply:
Because  of `bash`, `bash` required in order to work with lvg

Reply:
@rkiselenko there is a CVE associated with the `bash` package - https://ubuntu.com/security/CVE-2019-18276. We need to make a note here why we need to have `bash` and create a separate task to address CVE.